### PR TITLE
chore: use hereby instead of internal job

### PR DESCRIPTION
**Pull request by jrandolf-google**
Hereby is a simple task runner. It allows us to move the complex build commands into a file.

### CODE REVIEW

**Review by OrKoN — CHANGES_REQUESTED** [review]
can we use Promise.all and Node for this sort of thing? I am not sure why we needed the jobs in the first place as we don't have complex dependencies. Perhaps you could elaborate on that?

**Comment by jrandolf** [comment]
@OrKoN Hereby is a very minimal framework that does precisely that. Otherwise, we have to create scripts everywhere and maintain them all separately.

File: packages/puppeteer-core/tools/generate_sources.ts [hunk 1]
```diff
@@ -1,88 +0,0 @@
-#!/usr/bin/env node
-import {mkdir, mkdtemp, readFile, rm, writeFile} from 'fs/promises';
-import path, {join, resolve} from 'path';
-import {chdir} from 'process';
-
-import esbuild from 'esbuild';
-
-import {job} from '../../../tools/internal/job.js';
```
Comment (OrKoN) [inline packages/puppeteer-core/tools/generate_sources.ts:8]: is there anything else that uses tools/internal/job.ts?

Comment (jrandolf) [reply packages/puppeteer-core/tools/generate_sources.ts:8]: Yea, I plan to migrate them in another PR.

**Review by OrKoN — APPROVED** [review]

**Pull request merged**